packages/datagateway-common/src/detailsPanels/isis/datafileDetailsPanel.component.test.tsx

```diff
@@ -31,22 +50,65 @@ describe('Datafile details panel component', () => {
       createTime: '2019-06-11',
       description: 'Test description',
     };
-
-    (useDatafileDetails as jest.Mock).mockReturnValue({
-      data: rowData,
+    axios.get = jest.fn().mockResolvedValue({
+      data: [rowData],
     });
   });
 
   afterEach(() => {
     jest.clearAllMocks();
   });
 
-  it('renders correctly', () => {
-    const wrapper = createWrapper();
-    expect(wrapper.find('DatafileDetailsPanel').props()).toMatchSnapshot();
+  it('should render correctly', () => {
+    const { asFragment } = renderComponent({ rowData });
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should show default tab on first render', async () => {
+    renderComponent({
+      rowData,
+    });
+
+    expect(
+      await screen.findByRole('tab', { name: 'datafiles.details.label' })
+    ).toHaveAttribute('aria-selected', 'true');
   });
 
-  it('renders parameters tab when present in the data', () => {
+  it('should let user switch between tabs', async () => {
+    rowData.parameters = [
+      {
+        id: 2,
+        stringValue: 'String test',
+        type: {
+          id: 3,
+          name: 'String parameter',
+          units: 'foo/s',
+          valueType: 'STRING',
+        },
+      },
+    ];
+
+    renderComponent({ rowData });
+
+    expect(
+      await screen.findByRole('tabpanel', { name: 'datafiles.details.label' })
+    ).toBeVisible();
+
+    // switch tab
+    await user.click(
+      await screen.findByRole('tab', {
+        name: 'datafiles.details.parameters.label',
+      })
+    );
+
+    expect(
+      await screen.findByRole('tabpanel', {
+        name: 'datafiles.details.parameters.label',
+      })
+    ).toBeVisible();
+  });
```

Comment:
And this one too actually, I only see it in a few

Reply:
I think they are already in all the details panels that have tabs
- isis/dataset
- isis/datafile
- isis/instrument
- isis/investigation
- dls/dataset
- dls/visit

so this can be resolved